modules/ags/src/main/scala/lucuma/ags/Ags.scala

```diff
@@ -104,7 +104,8 @@ object Ags {
              guideStar,
              guideSpeed,
              quality,
-             NonEmptyList.one((position.posAngle, vignettingArea(gsOffset)))
+             position.posAngle,
+             vignettingArea(gsOffset)
```

Comment:
Is this going to be the vignetting for the one best position?

Reply:
After sortUsablePositions, it is the vignetting for the worst offset for that posAngle. But, it is the posAngle with the best vignetting (ie. it has the smallest worst vignetting).